Added Perigee Keyboard (#17212)
qmk/qmk_firmware: keyboards/keebmaker/perigee/readme.md

# Perigee

> ![Perigee]

Reviewer: There should be an image/photo here representative of the board.

```
![Perigee](https://imgur.com/someURL.jpg)
```

Author: All concerns have been addressed

> ![Perigee](https://imgur.com/a/wZjrkkx)

Reviewer 2: ```suggestion
![Perigee](https://i.imgur.com/Uzxhmujh.jpeg)
```

A 50% (12x5) ortholinear keyboard kit made and sold by KeebMaker LLC. 

* Keyboard Maintainer: [Kyle Lee](https://github.com/klee813)
* Hardware Supported: Perigee PCB rev1
* Hardware Availability: [keebmaker.com]

Make example for this keyboard (after setting up your build environment):

>     make perigee/rev1:default

Reviewer: ```suggestion
    make keebmaker/perigee/rev1:default
```

Flashing example for this keyboard:

>     make perigee/rev1:default:flash

Reviewer: ```suggestion
    make keebmaker/perigee/rev1:default:flash
```

> Perigee
> ===
> 
> ![Perigee]
> 
> A 50% (12x5) ortholinear keyboard kit made and sold by KeebMaker LLC. 
> 
> Keyboard Maintainer: [Kyle Lee](https://github.com/klee813)  
> Hardware Supported: Perigee PCB rev1  
> Hardware Availability: [keebmaker.com]
> 
> Make example for this keyboard (after setting up your build environment):
> 
>     make perigee/rev1:default
> 
> Install examples:
> 
>     make perigee/rev1:default:dfu         # For Perigee rev1
>     
> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

Reviewer 2: ```suggestion
# Perigee

![Perigee]

A 50% (12x5) ortholinear keyboard kit made and sold by KeebMaker LLC. 

* Keyboard Maintainer: [Kyle Lee](https://github.com/klee813)
* Hardware Supported: Perigee PCB rev1
* Hardware Availability: [keebmaker.com]

Make example for this keyboard (after setting up your build environment):

    make perigee/rev1:default

Flashing example for this keyboard:

    make perigee/rev1:default:flash
    
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Missing image.
